httpserver-rs/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wasmcloud-provider-httpserver"
-version = "0.1.3"
+version = "0.1.4"
```

Comment:
In the automation PR, I changed the version to `0.14.0` in order to be a minor change over the previously released httpserver, 0.13.0. I think this can be changed to `0.14.1` to stick with that